staging/src/k8s.io/apimachinery/pkg/util/mergepatch/errors.go

```diff
@@ -35,6 +36,12 @@ func ErrBadArgType(expected, actual string) error {
 	return fmt.Errorf("expected a %s, but received a %s", expected, actual)
 }
 
+func ErrBadArgTypeAssertion(expected, actual interface{}) error {
```

Comment:
Wait, there are only a couple left? Can't you just change that couple and remove the function once and for all?